### ISSUE TITLE
st_drivers missing?

### ISSUE THREAD

**Issue by marklit** [duckdblabs/duckdb_spatial#100]
```bash
$ ~/Downloads/duckdb_spatial/build/release/duckdb -unsigned
```

```
-- Loading resources from /home/mark/.duckdbrc
v0.8.2-dev1482 cc60796b7a
Enter ".help" for usage hints.
```

```sql
D SELECT * FROM st_drivers();
Error: Catalog Error: Table Function with name st_drivers does not exist!
Did you mean "range"?
LINE 1: SELECT * FROM st_drivers();
                      ^
```

```sql
D SELECT * FROM st_list_drivers();
Error: Catalog Error: Table Function with name st_list_drivers does not exist!
Did you mean "test_all_types"?
LINE 1: SELECT * FROM st_list_drivers();
                      ^
D
```

**Comment by Maxxen** [comment]
Building extensions "statically" has recently changed while I'm on vacation, none of the spatial functions will work when building latest duckdb directly from the extension repo. I'll look into It next week when I'm back.

**Comment by marklit** [comment]
Okay cool. Enjoy your holiday dude.

**Comment by chuckwondo** [comment]
@marklit, at the duckdb prompt, this works for me:

```plain
D install spatial;
D load spatial;
D select * from st_drivers();
┌────────────────┬──────────────────────┬───┬──────────┬──────────────────────┐
│   short_name   │      long_name       │ … │ can_open │       help_url       │
│    varchar     │       varchar        │   │ boolean  │       varchar        │
├────────────────┼──────────────────────┼───┼──────────┼──────────────────────┤
│ ESRI Shapefile │ ESRI Shapefile       │ … │ true     │ https://gdal.org/d…  │
...
├────────────────┴──────────────────────┴───┴──────────┴──────────────────────┤
│ 51 rows (40 shown)                                      6 columns (4 shown) │
└─────────────────────────────────────────────────────────────────────────────┘

**Comment by Maxxen** [comment]
This should be fixed now since #104
